docs/Edit_Page.md

```diff
@@ -0,0 +1,36 @@
+# Edit page
+## Link 
+mwet.lbl.gov/pages/\<specific_uid_here\>
+
+## Purpose
+This page displays a specific document and its corresponding references for the purpose of viewing and editing. 
+
+## Title
+The title of the document will be at the top of the page. You can edit the title by double clicking on it, or clicking the pencil icon. 
+
+## Metadata
+In the upper right corner you can see a card that will display the date created and the creator. If the document has been edited then it will also display the date it was last edited at and the name of the person who edited it.
+
+## View Past Versions
+This button will take you to the ["Versioned view"]() of this page.
```

Comment:
this link is empty...do you intend for it to a reference that you will be writing?